IdentityCore/src/controllers/broker/MSIDSSOExtensionInteractiveTokenRequestController.m

```diff
@@ -52,18 +53,54 @@ - (void)acquireToken:(MSIDRequestCompletionBlock)completionBlock
     {
         MSID_LOG_WITH_CTX(MSIDLogLevelInfo, self.requestParameters, @"Interactive broker extension flow finished. Result %@, error: %ld error domain: %@", _PII_NULLIFY(result), (long)error.code, error.domain);
         
+        if ([self shouldFallback:error])
```

Comment:
nit: small optimization if you do if (error && [self shouldFallback:error]). That way you don't even call below method if there's no error and don't check fallbackController presence.

Reply:
done